### ISSUE TITLE
Identifier/Accession number of the work of art depicted in the photo

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#7]
line 65: Identifier/Accession number of the work of art depicted in the photo 
Does Zeri have accession numbers for the works of art?

**Comment by marilenadaquino** [comment]
Yes, as you pointed out in line 106, there is a mistake there (and I see others followed my wrong example :P ). The value in line 106 should go in line 65. Do you want to change it for everybody or I should?

**Comment by edgartdata** [comment]
Yes, that would be great. thanks.
Can you please email me a blurb about that so that i can add it to my list of updates to share with the group?
Unfortunately there were rows added so the line numbers have changed :( I am open to a system to keep track of the fields without referring to line number if you can think of one.

**Comment by edgartdata** [comment]
I don't see Zeri values in either fields 65 or 106 now. What happened?

**Comment by edgartdata** [comment]
Never mind. I moved your artwork accession number values from line 112 to 68.

LDCS[@etichetta="Precise location"] | inventario 1890, n. 8358
-- | --

**Comment by edgartdata** [comment]
Do you want to indicate it as searchable?

**Comment by marilenadaquino** [comment]
I have not touched the spreadsheet for a long time (indeed the idea was not to mess around).. I think Francesca has not included the accession number in the wish list.
